pam/integration-tests/testdata/TestCLIAuthenticate/golden/authenticate_user_with_qr_code_in_a_tty

```diff
@@ -276,6 +294,9 @@ Scan the qrcode or enter the code in the login page
 ██████████████████████████████████████████████████████████████████
                                1337
 
+PAM Authenticate() for user "user-integration-qr-code-tty" exited with success
```

Comment:
Yeah, I think due to the lack of vertical space because of the bigger qrcode.

Sadly we didn't catch it in #393, but I noticed it here since I saw that changing the button label wasn't respected by these golden files. And it was just because there was not enough vertical space for the button and error messages to show, as written on the [commit](https://github.com/ubuntu/authd/commit/37f6d560e115a85a78495a18f56fa5629b139f73) that this change belongs to.

Reply:
ack then!